core/drivers/crypto/crypto_api/include/drvcrypt_acipher.h

```diff
@@ -133,6 +133,17 @@ struct drvcrypt_secret_data {
 	struct drvcrypt_buf secret; /* Shared secret */
 };
 
+/*
+ * Cipher data
+ */
+struct drvcrypt_cipher_data {
```

Comment:
Could you rename `drvcrypt_cipher_data` to `drvcrypt_acipher_data` ?

Reply:
In my opinion, acipher is the general name of many algorithms including ecc, and here specifically refers to ecc encryption and decryption. In addition, there are no similar names in other asymmetric encryption and decryption structures, such as rsa. Therefore, I think it is inappropriate to reflect the asymmetry here.

Reply:
Would `struct drvcrypt_ecc_ed` make sense? I'm thinking of the corresponding `struct drvcrypt_rsa_ed` above.

Reply:
I have renamed drvcrypt_cipher_data to struct drvcrypt_ecc_ed.